backends/discord/discord.cpp

```diff
@@ -0,0 +1,72 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#ifdef USE_DISCORD
+
+#define FORBIDDEN_SYMBOL_EXCEPTION_time_h
+#include "backends/discord/discord.h"
+#include "common/encoding.h"
+#include "common/translation.h"
+
+#include <discord_rpc.h>
+#include <time.h>
+
+#define DISCORD_CLIENT_ID "714287866464698470"
```

Comment:
Where does this come from? Is it kind of secret? If so, we probably could regenerate it and put at compile time, not from the source.

Reply:
This comes from the Discord Developer Portal, so it can't be regenerated. Is there a better place to put it?

Here's an example from another OSS: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/UICommon/DiscordPresence.cpp#L195